ompi/mca/osc/rdma/osc_rdma_accumulate.c

```diff
@@ -133,6 +134,28 @@ static int ompi_osc_rdma_op_mapping[OMPI_OP_NUM_OF_TYPES + 1] = {
     [OMPI_OP_REPLACE] = MCA_BTL_ATOMIC_SWAP,
 };
 
+/* set the appropriate flags for this atomic */
+static inline int ompi_osc_rdma_set_btl_flags(ompi_osc_rdma_module_t *module, ompi_datatype_t *dt, ptrdiff_t extent) {
+
+    int flags = 0;
+
+    if(4 == extent) {
+        flags = MCA_BTL_ATOMIC_FLAG_32BIT;
+
+    }
+
+    if(OPAL_UNLIKELY((module->acc_use_lock_fallback == true) || (module->disp_unit % extent))) {
```

Comment:
We try to put the constant first (as you did it in opal/mca/btl/base/btl_base_am_rdma.c)